From f2368f0aeac3ed1f359dd380d5f4ce3dcb4f89cd Mon Sep 17 00:00:00 2001
From: Patrick Delaunay <patrick.delaunay@foss.st.com>
Date: Tue, 12 Apr 2022 13:59:25 +0200
Subject: [PATCH 203/954] refactor(st-bsec): move defines in driver

Simplify the BSEC API file by moving the defines for IP configuration
in driver.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 874aef0babd265c5ab3ad4cb1e35df8e2099e025 refactor(st-bsec): move defines in driver]
Change-Id: Id134e4375a7b52004e917d0b6ec0db4666a1353e
Signed-off-by: Patrick Delaunay <patrick.delaunay@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/250944
Reviewed-by: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Yann GAUTIER <yann.gautier@foss.st.com>
Tested-by: Yann GAUTIER <yann.gautier@foss.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
---
 drivers/st/bsec/bsec2.c   | 7 +++++++
 include/drivers/st/bsec.h | 7 -------
 2 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/drivers/st/bsec/bsec2.c b/drivers/st/bsec/bsec2.c
index 397d5c9286..ef7ce39384 100644
--- a/drivers/st/bsec/bsec2.c
+++ b/drivers/st/bsec/bsec2.c
@@ -21,6 +21,13 @@
 #define BSEC_IP_VERSION_2_0	U(0x20)
 #define BSEC_IP_ID_2		U(0x100032)
 
+/*
+ * IP configuration
+ */
+#define BSEC_OTP_MASK			GENMASK(4, 0)
+#define BSEC_OTP_BANK_SHIFT		5
+#define BSEC_TIMEOUT_VALUE		U(0xFFFF)
+
 #define OTP_ACCESS_SIZE (round_up(OTP_MAX_SIZE, __WORD_BIT) / __WORD_BIT)
 
 static uint32_t otp_nsec_access[OTP_ACCESS_SIZE] __unused;
diff --git a/include/drivers/st/bsec.h b/include/drivers/st/bsec.h
index 507a1ac788..fc9d3d6ac1 100644
--- a/include/drivers/st/bsec.h
+++ b/include/drivers/st/bsec.h
@@ -12,13 +12,6 @@
 
 #include <lib/utils_def.h>
 
-/*
- * IP configuration
- */
-#define BSEC_OTP_MASK			GENMASK(4, 0)
-#define BSEC_OTP_BANK_SHIFT		5
-#define BSEC_TIMEOUT_VALUE		0xFFFF
-
 /*
  * Return status
  */
-- 
2.53.0

